src/MunicipalityRegistry.Projector/Caches/CachesController.cs

```diff
@@ -0,0 +1,50 @@
+namespace MunicipalityRegistry.Projector.Caches
+{
+    using System;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Be.Vlaanderen.Basisregisters.Api;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.LastChangedList;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.Extensions.Configuration;
+
+    [ApiVersion("1.0")]
+    [ApiRoute("caches")]
+    public class CachesController : ApiController
+    {
+        private readonly int _maxErrorCount;
+        private readonly int _maxErrorTimeInSeconds;
+
+        public CachesController([FromServices] IConfiguration configuration)
```

Comment:
Let's get rid of this entire ctor and add `[FromServices] IConfiguration configuration` to `Get()`, then you can get rid of the private fields too